### ISSUE TITLE
Grammar Inconsistency

### ISSUE THREAD

**Issue by Opalevanescence** [metabase/metabase#12694]
**Describe the bug**
When saving permissions, the "Save Changes" button has a capital C. Everywhere else in the site it's lowercase.

**To Reproduce**
1. Go to '/admin/permissions/databases'
2. Click on data access for one of your groups (either revoking or granting access)
3. In the upper right-hand corner, the save button says "Save Changes, but it should say "Save changes" for website consistency.

**Expected behavior**
Expected the C to be lowercase 😅

**Screenshot**
<img width="1120" alt="Screen Shot 2020-06-12 at 3 13 20 PM" src="https://user-images.githubusercontent.com/58329466/84550368-42f05880-acbf-11ea-9cdf-976579168c87.png">

**Severity**
So mild I'm not sure it's worth mentioning, but I'm a word nerd

**Comment by linzichao** [comment]
I can take it. 😊

**Comment by nemanjaglumac** [comment]
This string is one of **many** examples of similar inconsistencies. Fixing it in one place without systematically approaching the "problem" doesn't solve much.
Few more things to note:
- Cypress tests rely heavily on finding elements by their text content and need to be updated across the board
- This is a template string - it is used in translation files which themselves need to be updated after proposed change

To sum it up: we need to tackle this with a good naming/casing strategy, rather than fixing string by string.
